### DISCUSSION TITLE
OpenTelemetry Traces and Instrumentation

### DISCUSSION THREAD

**Discussion by Starefossen** [Unleash/unleash#6455] [Ideas]
Cross posting for visibility. I have attempted to use the OpenTelemetry Auto-instrumentation for Node.js in order to get trace data from Unleash to gain better insights into the operations of Unleash with varying degree of success.

Here is the guide for instrumenting Node.js applications I have followed: https://opentelemetry.io/docs/languages/js/automatic/

Here is the issue raised with the OpenTelemetry Node.js SDK https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1950

**Comment by daveleek** [comment]
Hello @Starefossen! Thank you for alerting us. Do you have what you need from Unleash for this? Any questions we can help answer?

**Reply by Starefossen** [reply]
The issue on https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1950 is still open, so I would assume this is still a problem, I have not tested in some weeks now. I still would like to get trace data from Unleash so hoping this will work at some point 🤞🏻

**Reply by daveleek** [reply]
Would you mind sharing a little about what you're working to achieve? Are you attempting to connect any of this to external (outside of Unleash) processes, and if so; in which context?
